docs/administration.index.md

```diff
@@ -3,32 +3,31 @@ id: administration.index
 title: Administration Roles
 ---
 
-Partners Users have access to the so called **Administration Area**. The administration area enables partner users to manage their child [meshWorkspace](./meshcloud.workspace.md) accounts within the meshcloud platform.
-Assigning Partner Users can only be done by Partner Admins.
+admin users have access to the so called **Administration Area**. The administration area enables admin users to manage their [workspace](./meshcloud.workspace.md) accounts within the meshcloud platform.
```

Comment:
f: this is the beginning of the sentence therefore the first letter should be in upper case
```suggestion
Admin users have access to the so called **Administration Area**. The administration area enables admin users to manage their [workspace](./meshcloud.workspace.md) accounts within the meshcloud platform.
```